README.md

```diff
@@ -93,5 +94,111 @@ When you have any questions, find a bug or have an idea for a feature you can cr
 
 ---
 
-Feel free to reach out, report issues, or share your ideas by creating an issue here on GitHub.  
+Feel free to reach out, report issues, or share your ideas by creating an issue [here](https://github.com/Dlurak/dlool_frontend_v2/issues) on GitHub.  
 Enjoy using Dlool!
+
+---
+---
+---
+---
+---
+
+# Deutsch
```

Comment:
Great you've created a new file for the german translation! But please remove the german part from the main README afterall we already have a link at the top